static/js/alert_words_ui.js

```diff
@@ -88,30 +87,50 @@ function remove_alert_word(alert_word) {
     });
 }
 
+export function show_add_alert_word_modal() {
+    const html_body = render_add_alert_word();
+
+    function add_alert_word_post_render() {
+        const $add_user_group_input_element = $("#add-alert-word-name");
+        const $add_user_group_submit_button = $("#add-alert-word .dialog_submit_button");
+        $add_user_group_submit_button.prop("disabled", true);
+
+        $add_user_group_input_element.on("input", () => {
+            $add_user_group_submit_button.prop(
+                "disabled",
+                $add_user_group_input_element.val() === "",
```

Comment:
Can we disable the button even when the input is only some number of spaces?

Reply:
Done.